src/components/NewPlantForm.js

```diff
@@ -1,17 +1,68 @@
-import React from "react";
+import React, { useState } from "react"
+
+function NewPlantForm({ originalPlants, setPlants }) {
+  const [formData, setFormData] = useState({
+    name: "",
+    image: "",
+    price: null,
+  })
+
+  function handleOnChange(event) {
+    setFormData({
+      ...formData,
+      [event.target.name]: event.target.value,
+    })
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault()
+    const plantData = {
+      name: formData.name,
+      image: formData.image,
+      price: formData.price,
+    }
+    fetch("http://localhost:6001/plants", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(plantData),
+    })
+      .then((r) => r.json())
```

Comment:
Not that it's wrong, but I will almost always prefer being explicit over implicit. 
```suggestion
      .then((response) => response.json())
```